server/src/main/java/org/elasticsearch/common/io/stream/StreamOutput.java

```diff
@@ -787,8 +787,10 @@ public final void writeOptionalInstant(@Nullable Instant instant) throws IOExcep
         })
     );
 
+    public static final byte GENERIC_LIST_HEADER = (byte) 7;
```

Comment:
We were already using this impl. detail in one spot, might as well just make it a constant here for now.